client/performance-profiler/pages/weekly-report/unsubscribe.tsx

```diff
@@ -10,6 +11,12 @@ import {
 import LoaderText from 'calypso/performance-profiler/components/weekly-report/loader-text';
 import { WeeklyReportProps } from 'calypso/performance-profiler/types/weekly-report';
 
+const Subtitle = styled.span`
+	color: #a7aaad;
+	font-size: 14px;
+	font-weight: 400;
+`;
+
```

Comment:
Should we reuse the `secondary-message` class [from the current implementation](https://github.com/Automattic/wp-calypso/blob/trunk/client/performance-profiler/components/message-display/style.scss#L99-L103) instead of creating a new styled element?

Reply:
That's much better, thanks for letting me know!

Reply:
Updated here https://github.com/Automattic/wp-calypso/pull/95227/commits/e67ff6504c016c2d8920baece4b7a8bfdd4cd35f